content-src/lib/snippets.js

```diff
@@ -62,10 +62,6 @@ export class SnippetsMap extends Map {
     }
   }
 
-  disableOnboarding() {
```

Comment:
Can you just have this be a noop? It will be removed when snippets gets removed. See other comment about leaving snippets alone